www/app/Panels/DockerContainersPanel.php

```
@@ -0,0 +1,370 @@
+        $project = $params['project'] ?? '';
+        $workingDir = $params['working_dir'] ?? '';
+
+        if (str_starts_with($project, 'pocket-dev')) {
+            return ['error' => 'Cannot stop PocketDev containers'];
+        }
+
+        if (empty($workingDir) || !is_dir($workingDir)) {
+            return ['error' => "Working directory not found: '{$workingDir}'"];
+        }
+
+        $result = Process::timeout(60)
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)